HomeWorkIntro/ThirdTask/main.css

```diff
@@ -13,47 +13,46 @@ body{
   background-color: #F5F5F5;
 }
 
-#top{
+.top{
   background-color: #800080;
-  left:0px;
-  top: 0px;
-  right:0px;
-  height: 130px;
+  width: 100%;
+  height: 15vh;
+  display: block;
```

Comment:
width 100% and display block are default properties for div element, you can remove them.